UEBPrep/app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,15 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
```

Comment:
@galxy25 This started out as a stub for authentication and session tracking. It should get overwritten when Devise.